requirements.txt

```diff
@@ -1,5 +1,6 @@
 ansible>=2.2
-six
+click
 pyOpenSSL
-PyYAML
-ruamel.yaml
+# We need to disable ruamel.yaml for now because of test failures
+#ruamel.yaml
```

Comment:
Test fixes that work for both PyYAML and ruamel.yaml will be handled in a followup PR.